rules-tests/TypeDeclaration/Rector/ClassMethod/AddMethodCallBasedStrictParamTypeRector/Fixture/skip_phpdoc_shape.php.inc

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddMethodCallBasedStrictParamTypeRector\Fixture;
+
+final class SkipPhpdocArrayShape {
+    private function doFoo() {
+        $shape = $this->returnsShape();
+        $this->doBar($shape['hello']);
+    }
+
+    /**
+     * @return array{'hello': string, 'world': int}
+     */
+    private function returnsShape() {
```

Comment:
for note, if you have native return `: array` type defined combined with docblock, then it is what it is :), as you got what you defined

Reply:
oh, as it detect by native type of ArrayDimFetch var, it seems already covered :)